devicetypes/qubino/qubino-3-phase-meter.src/qubino-3-phase-meter.groovy

```diff
@@ -185,11 +188,22 @@ private getMeterId(deviceNetworkId) {
 }
 
 private childRefresh(deviceNetworkId) {
-	def meterId = getMeterId(deviceNetworkId)
+	def meterId = getMeterId(deviceNetworkId) + 1
 	if (switchId != null) {
```

Comment:
where is `switchId` defined?

Reply:
Well, nowhere... But at least now it isn't used anywhere.